src/styles/_variables.scss

```diff
@@ -27,11 +27,44 @@ $metadata-modal-breakpoint: 700px;
 $global-toolbar-width: 80px;
 $run-list-controls-height: 95px;
 
-// -- levels -- //
-$z-index-metadata-panel: 3;
-$z-index-metadata-code: 3;
-$z-index-MuiTreeItem-icon: 1;
-$z-index-banner: 6;
+// z-index values
```

Comment:
This is nice, can we organize it by levels by grouping all with the at the same level?